packages/react-ui/src/widgets/FormulaWidgetUI.js

```diff
@@ -44,12 +44,13 @@ FormulaWidgetUI.defaultProps = {
 FormulaWidgetUI.propTypes = {
   data: PropTypes.oneOfType([
     PropTypes.string,
+    PropTypes.number,
     PropTypes.shape({
       value: PropTypes.oneOfType([
         PropTypes.number,
         PropTypes.string
-      ]).isRequired,
-      unit: PropTypes.string.isRequired
+      ]),
+      unit: PropTypes.string
```

Comment:
same here, until we want to allow empty units using this format, in that case, ok